webpack/stories/docs/api-middleware.stories.mdx

```diff
@@ -150,3 +150,26 @@ dispatch(
   })
 );
 ```
+
+## Update resources object
+
+After an API request SUCCESS action, it is often reconfirmed with a second GET request to view the updated data, and keep the Redux store up to date.
+However, if we receive a SUCCESS it is acceptable to assume that the data was actually updated as requested, and not completely necessary to make the additional request.
+To facilitate this, the API middleware can accept an `updatedData` object, with the updated resource object corresponding to its key.
+Thus, Redux's copy of the backend data is changed based on this trust, and an additional API request to show the updated data is avoided.
+
+> ⚠️ **Warning**: Please be aware that the given `updatedData` should be corresponding with the API call.
+
+Example:
+
+```js
+// This action creates a post API call and only if it ends succesfully,
+// the response object will update with the given 'updatedData' object
+dispatch(
+  post({
+    key: MY_SPECIAL_KEY,
+    url,
+    updatedData: {results: ['altered resource']}
```

Comment:
Why do we still send the data to the UPDATE action? Can't we get the updated data from the API response?

Reply:
AFAIK post requests (which alter a state) don't have to return the updated resource (i.e forget_status action) so I left it controlled by the consumer.

Reply:
no, they don't need to, but they usually do. I was talking about it in the meeting, we should state that returning the altered resource is a standard and only if we have a real case where we don't we would allow manual updates, but the updatedData seems very wrong.
If you insist on allowing this, I strongly believe you shouldn't be able to update all the data through that, but only the data that got updated, so it should be pair of `id` and `data` where data is data only for the given id.